tests/Auth0.AuthenticationApi.IntegrationTests/HttpClientAuthenticationConnectionTests.cs

```diff
@@ -13,7 +13,7 @@ public async Task Disposes_HttpClient_it_creates_on_dispose()
         {
             var connection = new HttpClientAuthenticationConnection();
             connection.Dispose();
-            await Assert.ThrowsAsync<ObjectDisposedException >(() => connection.GetAsync<string>(new Uri("https://" + GetVariable("AUTH0_AUTHENTICATION_API_URL"))));
+            await Assert.ThrowsAsync<ObjectDisposedException >(() => connection.GetAsync<string>(new Uri("https://www.auth0.com")));
```

Comment:
Interesting ... Previously, we were pinging `https://auth0-dotnet-integration-tests.auth0.com`, which works fine. Now our tenant uses a different URL: `https://dx-sdks-testing.us.auth0.com` which doesn't work. Apparently, there is different behavior when ur URL tenant contains a regional subdomain (eu, us, ...).

Therefor I updated this to just ping our public website instead.

Reply:
Additionally, do you feel that testing whether `HttpClient` is disposed or not, is a valid or useful test?

Reply:
Yes, I think it makes sense, to avoid leaks sneaking in in future PRs.